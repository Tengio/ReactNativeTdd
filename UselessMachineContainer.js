import React, { Component } from 'react';
import { Text, View } from 'react-native';
import CustomButton from './CustomButton';

export default class UselessMachineContainer extends React.Component {
    render() {
        return <View>
                <Text>Useless machine</Text>
                <CustomButton />
                <CustomButton />
                </View>
    }
}
