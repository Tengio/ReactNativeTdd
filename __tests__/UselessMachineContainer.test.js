import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import UselessMachineContainer from '../UselessMachineContainer';

describe('UselessMachineContainer', () => {
    it('should render two button and a text', () => {
        const tree = renderer.create(
            <UselessMachineContainer/>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
})