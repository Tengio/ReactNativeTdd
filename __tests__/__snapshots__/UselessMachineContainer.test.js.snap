// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UselessMachineContainer should render two button and a text 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Useless machine
  </Text>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    />
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    />
  </View>
</View>
`;
